### ISSUE TITLE
PROPOSAL - Auto-post late user submissions.

### ISSUE THREAD

**Issue by skyscooby** [standup-raven/standup-raven#94]
**Is your feature request related to a problem? Please describe.**
If a user submits their standup report after the daily report has been triggered. Others need to manually generate a second report to see that users contributions. 

**Describe the solution you'd like**
Anytime post the daily report being automatically triggered, have the users individual submission reports post directly to the channel.

**Describe alternatives you've considered**
We have adopted an internal rule that if you are late you must post your submission directly to the channel instead of using the bot itself.. this is less than ideal because people forget and results in two different workflows. Nobody sees their contributions unless they are constantly requesting the previous days report to see if the missing reports were submitted late.

**Comment by harshilsharma63** [comment]
@skyscooby thanks a lot for the feature proposal! This is something we've felt as well.

I'm trying to examine the various solutions here. How about updating the original report itself v/s adding a second, user-exclusive post?

**Comment by skyscooby** [comment]
@harshilsharma63 I believe that would work provided you also posted a new message in the channel saying "Todays report has been updated by user X" or some other way to signal to the group that they can do a targeted second look at one section of the report.. The reports can get quite long and re-reading the whole thing to figure out what area changed isn't ideal.

**Comment by harshilsharma63** [comment]
@skyscooby agreed. We need a way to either highlight the change in existing report, or post just the changes. I'll definitionally look into this and am open to all ideas.

**Comment by shieldsjared** [comment]
@harshilsharma63 - I'm happy to submit a PR for this issue :) Do you have a preference on how we go about this?  There are a few options on the table:

1. Updating existing report 
- Perhaps edit the existing report, and then _reply to_ it with something like "Report has been updated by @username" (the reply would satisfy @skyscooby's request, and by using a reply it'll notify users of the change)
2. Delete old report and post a new one, again, with the added text "Report has been updated by @username"...

Haven't really dug into the code yet to see if either of those will be more _convenient_ to implement... that aside, is there a preference for which path to go down?
